apiary.apib

```diff
@@ -1456,6 +1456,11 @@ For example you should create a group for the Introduction to Programming course
         + meta_data (object, optional) - You can use this parameter to attach key-value strings to the object. Meta data is useful for storing additional, structured information on an object.
         + learning_outcomes(array, optional) - Searchable keywords identifying skills credential-holders need to demonstrate (or have learned) in the process of earning this credential.
         + generate_private_credential (boolean, optional) - The default value of `Credential.private` for the Group. The supported values are `true`, `false`, and `null`. When this is set to `true` or `false`, it will override `Department.generate_private_credential` and be used as the default of `Credential.private` when creating a new `Credential`. When this is set to `null`, it won't override `Department.generate_private_credential` which will be used for the default of `Credential.private` instead.
+        + auto_expiry - (number, optional) - If the Group's credentials expire, it's important to set an expiration period. Add a year to say how long the credentials for this group should be valid.
```

Comment:
Just wanted to double check, have you run these texts with Kelly?

Reply:
Yes, I'm waiting for confirmation before merging